### PR TITLE
Fix:Added Functionality for inviting users to Host Team (#477)

### PR DESCRIPTION
**Pull request by Ram81**
URL & method paramters are empty when the API for inviting user to host team will be ready we can put it !
#477 resolved, please review!

### CODE REVIEW

**Comment by deshraj** [comment]
@Ram81 added the API. Now, you can make changes accordingly.

**Comment by Ram81** [comment]
@deshraj  in the PR #504 I think @anweshknayak has also added functionality for inviting user to host team, can you please check that

**Comment by Ram81** [comment]
@deshraj  : changes done please review!

**Comment by deshraj** [comment]
@Ram81 lets remove the debug messages from console and we are good to go.

**Comment by Ram81** [comment]
@deshraj  is this Okay?

**Comment by deshraj** [comment]
No, the nouns should be in plural form. We have missed it in some places but we will change those afterwards since it will affect the client side.

**Comment by Ram81** [comment]
@deshraj  please review.

**Comment by deshraj** [comment]
@aka-jain can you please review the PR?

File: frontend/src/js/controllers/challengeHostTeamsCtrl.js [hunk 1]
```diff
@@ -192,6 +192,44 @@
...
+                    parameters.token = userKey;
+                    parameters.callback = {
+                        onSuccess: function(response) {
+                            var status = response.status;
+                            var response = response.data;
+                        },
+                        onError: function(response) {
+                            var status = response.status;
+                            var error = response.data;
+                            console.log(error);
```
Comment (aka-jain) [inline frontend/src/js/controllers/challengeHostTeamsCtrl.js]: @Ram81 : Please remove console messages.

File: frontend/src/js/controllers/challengeHostTeamsCtrl.js [hunk 2]
```diff
@@ -192,6 +192,44 @@
...
+                        }
+                    };
+
+                    utilities.sendRequest(parameters);
+                }, function() {
+                    console.log("Operation Aborted");
```
Comment (aka-jain) [inline frontend/src/js/controllers/challengeHostTeamsCtrl.js]: @Ram81 : Remove this too.

**Comment by Ram81** [comment]
@aka-jain changes Done. Please review!

**Comment by aka-jain** [comment]
@Ram81 : Great Job.

**Pull request merged**